src/movement/thruster_sabertooth.cpp

```diff
@@ -84,12 +89,39 @@ void callBack(const robosub::thruster::ConstPtr& msg)
 
 }
 
+void setTimeOut (uint8_t ms_100)
+{
+  uint8_t serial_data[4];
+
+  //loop through 6 thrusters
+  for (int i = 0; i < 6; i++)
+  {
+    //adress: loop over addresses
+    serial_data[0] = mThruster_info[i].address;
+    //command
+    serial_data[1] = 14; //serial timeout
+    //timeout value
+    serial_data[2] = ms_100;
+    //checksum
+    checksum(serial_data);
+  }
+
+    //send package to thrusters
+    mSerial.Write(serial_data, 4);
+
+  //Publish thruster info with name
```

Comment:
This is not what this does. The comment is inaccurate. Also, it may be more useful to output when each thruster (and which one) is having its timeout set.